core/src/main/resources/defaults/blueprints/7.2.17/cdp-sdx-scalable.bp

```diff
@@ -0,0 +1,516 @@
+          {
+            "base":true,
+            "refName":"hbase-HBASETHRIFTSERVER-BASE",
+            "roleType":"HBASETHRIFTSERVER"
+          }
+        ],
+        "serviceConfigs":[
+          {
+            "name":"hbase_wal_dir",
+            "value":"/hbase-wals"
+          },
+          {
+            "name":"service_config_suppression_hadoop_secure_web_ui",
+            "value":"true"
+          }
+        ],
+        "serviceType":"HBASE"
+      },
+      {
+        "refName":"hdfs",
+        "roleConfigGroups":[
+          {
+            "base":true,
+            "refName":"hdfs-NAMENODE-BASE",
+            "roleType":"NAMENODE",
+            "configs":[
+              {
+                "name":"role_config_suppression_namenode_java_heapsize_minimum_validator",
+                "value":"true"
+              },
+              {
+                "name":"namenode_java_heapsize",
+                "value":1073741824
+              },
+              {
+                "name":"datanode_java_heapsize",
+                "value":3219128320
+              }
+            ]
+          },
+          {
+            "base":true,
+            "refName":"hdfs-SECONDARYNAMENODE-BASE",
+            "roleType":"SECONDARYNAMENODE"
+          },
+          {
+            "base":true,
+            "refName":"hdfs-DATANODE-BASE",
+            "roleType":"DATANODE",
+            "configs":[
+              {
+                "name":"dfs_datanode_max_locked_memory",
+                "value":1073741824
+              }
+            ]
+          },
+          {
+            "base":true,
+            "refName":"hdfs-BALANCER-BASE",
+            "roleType":"BALANCER"
+          },
+          {
+            "base":true,
+            "refName":"hdfs-GATEWAY-BASE",
+            "roleType":"GATEWAY"
+          },
+          {
+            "base":true,
+            "refName":"hdfs-FAILOVERCONTROLLER-BASE",
+            "roleType":"FAILOVERCONTROLLER"
+          },
+          {
+            "base":true,
+            "refName":"hdfs-JOURNALNODE-BASE",
+            "roleType":"JOURNALNODE"
+          }
+        ],
+        "serviceConfigs":[
+          {
+            "name":"enable_ranger_authorization",
+            "value":"true"
+          },
+          {
+            "name":"service_health_suppression_hdfs_verify_ec_with_topology",
+            "value":"true"
+          },
+          {
+            "name":"dfs_replication",
+            "value":"2"
+          },
+          {
+            "name":"service_config_suppression_datanode_count_validator",
+            "value":"true"
+          }
+        ],
+        "serviceType":"HDFS"
+      },
+      {
+        "refName":"kafka",
+        "roleConfigGroups":[
+          {
+            "base":true,
+            "refName":"kafka-GATEWAY-BASE",
+            "roleType":"GATEWAY"
+          },
+          {
+            "base":true,
+            "refName":"kafka-KAFKA_BROKER-BASE",
+            "roleType":"KAFKA_BROKER"
```

Comment:
kafka memory configuration required ?

Reply:
Why is it not required ?

Reply:
[CDPSDX-4005](https://jira.cloudera.com/browse/CDPSDX-4005)